packaging/pkg_helpers.bash

```diff
@@ -0,0 +1,133 @@
+# A set of useful bash functions for common functionality we need to do in
+# many build scripts
+
+# Respecting PYTHON_VERSION and UNICODE_ABI, add (or install) the correct
+# version of Python to perform a build.  Relevant to wheel builds.
+setup_python() {
+  if [[ "$(uname)" == Darwin ]]; then
+    eval "$(conda shell.bash hook)"
+    conda create -yn "env$PYTHON_VERSION" python="$PYTHON_VERSION"
```

Comment:
what if such an env is already created, and potentially corrupted?

Reply:
```
    conda env remove -n "env$PYTHON_VERSION" || true
```

:)